projects/adin1300_dual/common/adin1300_dual_qsys.tcl

```diff
@@ -0,0 +1,331 @@
+
+# Instances and instance parameters
+add_instance gmii_to_rgmii_adapter_0 altera_gmii_to_rgmii_adapter
+set_instance_parameter_value gmii_to_rgmii_adapter_0 {RX_PIPELINE_DEPTH} {1}
+set_instance_parameter_value gmii_to_rgmii_adapter_0 {TX_PIPELINE_DEPTH} {1}
+
+add_instance hps_emac_interface_splitter_0 altera_hps_emac_interface_splitter
+
+add_instance gmii_to_rgmii_adapter_1 altera_gmii_to_rgmii_adapter
+set_instance_parameter_value gmii_to_rgmii_adapter_1 {RX_PIPELINE_DEPTH} {1}
+set_instance_parameter_value gmii_to_rgmii_adapter_1 {TX_PIPELINE_DEPTH} {1}
+
+add_instance hps_emac_interface_splitter_1 altera_hps_emac_interface_splitter
+
+add_instance iopll_0 altera_iopll
+set_instance_parameter_value iopll_0 {gui_active_clk} {0}
+set_instance_parameter_value iopll_0 {gui_c_cnt_in_src0} {c_m_cnt_in_src_ph_mux_clk}
```

Comment:
Can we remove part of the default parameters ? I don't think we need to expose all parameters if only a few of them are different than the default values

Reply:
Removed some of the parameters.